Assets/Scripts/Inventory/InventoryItem_Equippable.cs

```diff
@@ -5,6 +5,8 @@
 [AddComponentMenu("Inventory/Item/Equippable")]
 public class InventoryItem_Equippable : InventoryItem
 {
+    public bool equipped = false;
```

Comment:
you are mixing different responsibilities here (SRP).
is this an item definition, or is it an item in-world, or is it an item in inventory or is it an equipped item. These are different things!